portfolio/src/main/java/com/google/sps/servlets/NumCommentsServlet.java

```diff
@@ -54,24 +56,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<Integer> numComments = new ArrayList<> ();
+    Map<Integer, Integer> numComments = new HashMap<Integer, Integer> ();
 
     // Receive input from the pagination to see which comments to show
     FetchOptions entitiesLimit = FetchOptions.Builder.withLimit(COMMENT_LIMIT);
     double totalComments = results.countEntities(entitiesLimit);
 
     if (totalComments > 0) {
-      int countSize = 0;
+      int currentNumber = 0;
       Iterable<Entity> entitiesList = results.asIterable();
       for (Entity entity : entitiesList) {
```

Comment:
Since we're dealing with both comments and blogs, it's not clear what `entitiesList` and `entity` refer to. Can you name them either commentList or blogPostList depending on what they contain?